firmware/controllers/sensors/converters/thermistor_func.cpp

```diff
@@ -25,6 +25,12 @@ SensorResult ThermistorFunc::convert(float ohms) const {
 
 	float celsius = convertKelvinToCelcius(kelvin);
 
+	// bounds check result - please don't try to run rusEfi when colder than -50C
+	// high end limit is required as this could be an oil temp sensor
+	if (celsius < -50 || celsius > 200) {
```

Comment:
in an ideal world I would have those magic constants more explicitly defined on top of the file, not in the middle of random